docs/using-gitbase/functions.md

```diff
@@ -25,6 +25,7 @@ These are all functions that are available because they are implemented in `go-m
 |:-------------|:-------------------------------------------------------------------------------------------------------------------------------|
 |`ARRAY_LENGTH(json)`|If the json representation is an array, this function returns its size.|
 |`AVG(expr)`|Returns the average value of expr in all rows.|
+|`BLAME(expr)`|Returns a list, as a json array, of lines changes and authorship.|
```

Comment:
This should be on gitbase functions, not here. This part of the documentation is auto-generated using go-mysql-server docu

Reply:
Also, blame is receiving two params: `BLAME(repository_id, commit_hash)`

Reply:
This is still here, should we still remove it and keep it in go-mysql-server?

Reply:
Moved.